nixos/modules/programs/nano.nix

```diff
@@ -12,16 +12,16 @@ in
 
       nanorc = lib.mkOption {
         type = lib.types.lines;
-        default = "";
+        default = ''
+          set nowrap
+          set tabstospaces
+          set tabsize 2
+          include "${pkgs.nano}/share/nano/*.nanorc"
```

Comment:
@globin For me this line is not added when I set `programs.nano.nanorc` in configuration.nix, or am I miss something?